package.json

```diff
@@ -3,9 +3,9 @@
   "version": "0.3.3",
   "main": "lib/index.cjs.js",
   "typings": "lib/index.d.ts",
-  "module": "lib/index.esm.js",
-  "unpkg": "lib/index.iffe.js",
-  "jsdelivr": "lib/index.iffe.js",
+  "module": "lib/index.es.js",
+  "unpkg": "lib/index.iife.js",
+  "jsdelivr": "lib/index.iife.js",
```

Comment:
Thanks for spotting out the `iife` typo, but I would like to keep the `esm` naming.